packages/node_modules/@webex/plugin-meetings/src/peer-connection-manager/index.js

```diff
@@ -84,6 +84,13 @@ const setMaxFs = (sdp, level = QUALITY_LEVELS.HIGH) => {
   return replaceSdp;
 };
 
+
+const setMaxFsOnRemoteSdp = (sdp) => {
```

Comment:
function name is wrong, the code is not setting maxFs but adding x-google-start-bitrate